module/batch-action/src/Application/Controller/Api/CreateBatchAction.php

```diff
@@ -106,7 +106,8 @@ public function __invoke(Request $request): BatchActionId
                     BatchActionId::generate(),
                     $actionType,
                     $filter,
-                    $data->payload ?: null
+                    $data->payload ?: null,
+                    $data->autoEndOnErrors ?: true
```

Comment:
Wouldn't it be cleaner to have default true in form model? More reusable that way.